Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Name your repository according to your chosen topic, this repository will be used for all your practical tasks for OOP subject, not only for practice 1